### DISCUSSION TITLE
Apple M1 testbed?

### DISCUSSION THREAD

**Discussion by baentsch** [open-quantum-safe/liboqs#947] [Ideas]
Would it be a good thing for OQS to have access to an M1 build&test env? Maybe also for benchmarking? @dstebila mentioned https://www.scaleway.com/en/hello-m1/ Good idea? Does anyone know better providers?

**Comment by dstebila** [comment]
I've just signed up for an account with Scaleway but only after signing up do they tell you that they are "out of stock" on Apple M1 machines. I've requested to be notified when they have more "in stock". :(

**Reply by baentsch** [reply]
"out of stock" in a cloud... gets me thinking what this means: Bright blue skies? :-) Anyway, thanks for the update.
